### PR TITLE
Filter section

### PR DESCRIPTION
**Pull request by NacymeG**

### CODE REVIEW

**Review by Caroline-CD — DISMISSED** [review]
Ok for me

**Review by Caroline-CD — APPROVED** [review]
It's ok

File: src/Containers/FilterOption.jsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  useEffect(() => {
+    axios.get(`https://api.magicthegathering.io/v1/sets`).then(({ data }) => {
+      setlistSets(data.sets);
+    });
+  }, []);
+  useEffect(() => {
+    axios.get(`https://api.magicthegathering.io/v1/types`).then(({ data }) => {
+      setlistTypes(data.types);
+    });
+  }, []);
```
Comment (LoicBrassart) [inline src/Containers/FilterOption.jsx:56]: You could have called both APIs in the same useEffect

File: src/Containers/FilterOption.jsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+          <option value="Uncommon">Uncommon</option>
+          <option value="Rare">Rare</option>
+          <option value="Mythic">Mythic</option>
+        </select>
+        Set :
+        <select name="sets" id="sets" onChange={HandlerChangeSetValue}>
+          <option value="">Select a Set</option>
+          {listSets.map((set) => {
+            return (
+              <>
```
Comment (LoicBrassart) [inline src/Containers/FilterOption.jsx:104]: Useless Frgment

**Review by LoicBrassart — APPROVED** [review]
No real Evil

**Pull request merged**